internal/pkg/deploy/pipeline.go

```diff
@@ -16,9 +16,14 @@ import (
 )
 
 const (
-	fmtInvalidRepo = "unable to locate the repository URL from the properties: %+v"
+	fmtInvalidRepo           = "unable to locate the repository URL from the properties: %+v"
```

Comment:
nit: maybe we can change the wording of this error msg slightly now that we have `fmtErrMissingProperty` because this err is only called when we can't parse the repo URL properly, right? 
```suggestion
	fmtInvalidRepo           = "unable to parse the repository from the URL %+v"
```